src/main/java/seedu/tp/commands/DeleteCommand.java

```diff
@@ -15,10 +21,20 @@
      * @param index         index in the delete command
      */
     public DeleteCommand(FlashcardList flashcardList, int index) {
+        setupLogger();
```

Comment:
I think that setupLogger needs to be called once when the program starts up, not every time we instnatiate a Command. logger is a static member so redoing the setup every time we create a new flashcard is wasteful. I think maybe we should have a method that runs at the start of the program that does set up of all the loggers. 

https://www.vogella.com/tutorials/Logging/article.html

We can see from this example that prof linked that they are calling MyLogger.setup() from main() in the last example code. What do u think?

Reply:
Agree :) I'll refactor the code to follow this example you shared. Thanks!